Jenkinsfile

```diff
@@ -81,6 +81,109 @@ pipeline {
+                        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+                        export NVM_DIR="$HOME/.nvm" 
+                        . $NVM_DIR/nvm.sh
+                        nvm i 10
+
+                        # Install docker-compose 
+                        curl -L https://github.com/docker/compose/releases/download/1.21.2/docker-compose-`uname -s`-`uname -m` -o ~/docker-compose
+                        chmod +x ~/docker-compose
+
+                        # Run eslint on portal code
+                        cd src/pfe/portal
+                        npm install
+                        npm run eslint
+                        if [ $? -ne 0 ]; then
+                            exit 1
+                        fi
+                        cd ../../..
+
+                        # Create codewind-workspace if it does not exist
+                        printf "\n\nCreating codewind-workspace\n"
+                        mkdir -m 777 -p codewind-workspace
+
+                        export REPOSITORY='';
+                        export TAG
+                        export WORKSPACE_DIRECTORY=$PWD/codewind-workspace;
+                        export HOST_OS=$(uname);
+                        export REMOTE_MODE;
+                        export HOST_HOME=$HOME
+                        export ARCH=$(uname -m);
+                        # Select the right images for this architecture.
+                        if [ "$ARCH" = "x86_64" ]; then
+                          export PLATFORM="-amd64"
+                        else
+                          export PLATFORM="-$ARCH"
+                        fi
+
+                        # Start codewind running
+                        ~/docker-compose -f docker-compose.yaml -f docker-compose-local.yaml up -d;
+
+                        if [ $? -eq 0 ]; then
+                            # Reset so we don't get conflicts
+                            unset REPOSITORY;
+                            unset WORKSPACE_DIRECTORY;
+                            unset REMOTE_MODE;
+                            printf "\n\n${GREEN}SUCCESSFULLY STARTED CONTAINERS $RESET\n";
+                            printf "\nCurrent running codewind containers\n";
+                            docker ps --filter name=codewind
+                        else
+                            printf "\n\n${RED}FAILED TO START CONTAINERS $RESET\n";
+                            exit;
+                        fi
+
+                        printf "\n\nPausing for 20 seconds to allow containers to start\n";
+                        sleep 20;
+
+                        # Check to see if any containers exited straight away
+                        printf "\n\n${BLUE}CHECKING FOR codewind CONTAINERS THAT EXITED STRAIGHT AFTER BEING RUN $RESET\n";
+                        EXITED_PROCESSES=$(docker ps -q --filter "name=codewind" --filter "status=exited"  | wc -l)
```

Comment:
Think we need the `-a` flag to find exited containers here and in the other uses of `docker ps` below?